README.md

```diff
@@ -41,6 +41,7 @@ Fields:
  - `oliName` Type: String. Required. Name of the bot personality. (To do: consider changing name. Though maybe we want to keep it for nostalgia resons.)
  - `abbr` Type: String. Optional. Abbreviated form of the institution's name.
  - `animal` Type: String. Optional. The species (or whatever) of the institution's mascot.
+ - `availableRegions` Type: \[String\]. Optional. List of regions to prepopulate into the region picker.
```

Comment:
Does this ultimately have filtering/permission implications? e.g. if the `brandedCollege` has `availableRegions` but the individual user doesn't have a `region`, what happens?

Reply:
They're considered national, as it works already. This is just to add values to the region picker in the addUser.